internal/pkg/earpointer/analysis.go

```diff
@@ -64,22 +64,25 @@ type visitor struct {
 
 func run(pass *analysis.Pass) (interface{}, error) {
 	ssainput := pass.ResultOf[buildssa.Analyzer].(*buildssa.SSA)
-	p := analyze(ssainput.Pkg)
+	p := Analyze(ssainput)
 	return p, nil
 }
 
 // Analyzes an SSA program and build the partition information.
-func analyze(pkg *ssa.Package) *Partitions {
-	prog := pkg.Prog
+func Analyze(ssainput *buildssa.SSA) *Partitions {
```

Comment:
If this is going to be the entry-point for the package, it's unclear to me what role the `earpointer.Analyzer` is intended to play.  If it is not going to be a "upstream" analyzer for `levee.Analyzer`, is it just a harness for testing?